modules/display.c

```diff
@@ -3873,6 +3883,80 @@ static void mdy_blanking_cancel_timers(void)
     //mdy_blanking_stop_adaptive_dimming();
 }
 
+/** After bootup autoblank prevent; assume not active */
+static int64_t mdy_blanking_afterboot_limit = 0;
+
+/** Get delay until end of after boot blank prevent
+ */
+static gint mdy_blanking_get_afterboot_delay(void)
```

Comment:
Could be useful to include the unit in the function name -- is it milliseconds or seconds or something else?